### PR TITLE
Treat .svelte file extensions just like .vue

### PR DESCRIPTION
**Pull request by jobelenus**

### CODE REVIEW

File: tests/parser-spec.ts [hunk 1]
```diff
@@ -633,6 +633,28 @@ describe('parsing', function() {
...
+        it('parses a svelte file without included files', function() {
+            const file = getFixturePath('svelte.svelte');
+            const comments = getComments(file, {
+                withInlineFiles: false,
+            });
+            comments.should.have.length(1);
+            verifyComment(comments[0], 'TODO', 2, 'Svelte script comment');
+        });
+
+        it('parses a vue file with just php', function() {
```
Comment (pgilad) [inline tests/parser-spec.ts]: This should be `svelte`

File: tests/parser-spec.ts [hunk 2]
```diff
@@ -633,6 +633,28 @@ describe('parsing', function() {
...
+            const file = getFixturePath('vue.vue');
```
Comment (pgilad) [inline tests/parser-spec.ts]: Should use the `svelte.svelte` fixture :smile:

**Comment by pgilad** [comment]
Great work 🏋 a little fixup for this to be perfect 😅

File: tests/parser-spec.ts (hunk 2 above)
Comment (jobelenus) [reply tests/parser-spec.ts]: Bah, didn't see it loaded twice. Should be all better now.

**Comment by pgilad** [comment]
Thanks!!!

**Comment by jobelenus** [comment]
Thank you!

**Pull request merged**